experimenter/autosklearn/pipelines.py

```diff
@@ -4,7 +4,8 @@
 
 from .primitives import AutoSklearnClassifierPrimitive
 
-def get_classification_pipeline():
+def get_classification_pipeline(time_limit=20):
```

Comment:
There is another hyper-parameter to limit the time per model. The default value is 10% of the default for the total time. Let's make that time limit always 10% of the total time limit.